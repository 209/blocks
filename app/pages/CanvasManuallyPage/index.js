/*
 * CanvasManuallyPage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 */

import React from 'react';
// import { FormattedMessage } from 'react-intl';
// import messages from 'pages/CanvasManuallyPage/messages';
import CanvasManuallyBoard from 'containers/CanvasManuallyBoard';

export default function CanvasManuallyPage() {
  return (
    <>
      <h2>Canvas Manually</h2>
      <CanvasManuallyBoard />
    </>
  );
}
