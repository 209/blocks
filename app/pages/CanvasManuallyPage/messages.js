/*
 * CanvasManuallyPage Messages
 *
 * This contains all the text for the CanvasManuallyPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.CanvasManuallyPage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the CanvasManuallyPage container!',
  },
});
