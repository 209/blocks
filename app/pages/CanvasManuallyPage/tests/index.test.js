import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import CanvasManuallyPage from 'pages/CanvasManuallyPage/index';

describe('<CanvasManuallyPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <CanvasManuallyPage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
