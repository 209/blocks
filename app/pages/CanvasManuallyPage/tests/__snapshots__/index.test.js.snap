// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CanvasManuallyPage /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the CanvasManuallyPage container!
  </span>
</h1>
`;
